src/screens/Home/Home.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { SafeAreaView, Text, ActivityIndicator } from 'react-native';
+import styles from './Home.style';
+import SearchBar from "../../components/SearchBar/SearchBar";
+import RepoList from "../../components/RepoList/RepoList";
+import useRepos from "../../hooks/useRepos";
+import useSearch from '../../hooks/useSearch';
+
+const Home = () => {
+    const [query, setQuery] = useState('');
+
+    const onSuccess = (data) => {
+        console.log('Successfully fetched')
+    }
+    const onError = (error) => {
+        console.log('Error encountered ', error.message)
+    }
+
+    const { data: initialData, isLoading: isLoadingInitial, isError: isErrorInitial } = useRepos(query, onSuccess, onError)
+    const { data: searchData, isLoading: isLoadingSearch, isError: isErrorSearch, 
+        refetch, fetchNextPage, hasNextPage, isFetchingNextPage, status } = useSearch(query, onSuccess, onError);
```

Comment:
We need to discuss the API calls on query change please when possible.
This is something I missed to tell you about before and I just noticed it.

Reply:
As discussed, nothing to do here, working fine.
Thanks.